index.html

```diff
@@ -3053,15 +3053,105 @@ <h2>Identifier</h2>
+          </thead>
+          <tbody>
+            <tr>
+              <td>specversion</td>
+              <td>string</td>
+              <td>mandatory, non-empty</td>
+              <td>The <code>specversion</code> MUST have the value "1.0"</td>
+            </tr>
+            <tr>
+              <td>id</td>
+              <td>string</td>
+              <td>mandatory, non-empty</td>
+              <td>For each distinct event the combination of <code>source</code> + <code>id</code> MUST be unique. 
+                A WebThing MAY resend the same event with the same <code>id</code>. 
+                If an event is resent with the same <code>id</code>, a consumer SHOULD treat it as
+                duplicate.</td>
+            </tr>
+            <tr>
+              <td>source</td>
+              <td>uri</td>
+              <td>mandatory, non-empty</td>
+              <td>The combination of <code>source</code> + <code>id</code> MUST be unique for each distinct event. 
+                The source MUST be set to the <code>baseURI</code> of the WebThing.</td>
+            </tr>
+            <tr>
+              <td>subject</td>
+              <td>string</td>
+              <td>optional</td>
+              <td>When this field is used, it MUST be set to the <code>title</code> of the event or property
+                affordance.</td>
+            </tr>
+            <tr>
+              <td>type</td>
+              <td>string</td>
+              <td>mandatory, non-empty</td>
+              <td>For event affordances the source MUST be set to the string "/events/<code>eventName</code>" of the
+                event affordance.</br>
+                For property affordances the source MUST be set to the string "/properties/<code>propertyName</code>" of
+                the property affordance.
+              </td>
+            </tr>
+            <tr>
+              <td>time</td>
+              <td>string</td>
+              <td>optional, compliant with [[RFC3339]]</td>
+              <td>time when the event occurred. If the WebThing does not have a clock, it MUST use the same
+                algorithm on all event affordances to determine the value of this field.
+            </tr>
+            <tr>
+              <td>datacontenttype</td>
+              <td>string</td>
+              <td>optional, [[RFC2046]] compliant MIME type, when set the value MUST be "<code>application/json</code>"</td>
+              <td>If a WebThing contains a <code>data member</code>, this field MUST be present.</td>
+            </tr>
+            <tr>
+              <td>data</td>
+              <td>string</td>
+              <td>optional</td>
+              <td>JSON encoded <code>data</code> object of the event affordance.</td>
+            </tr>
+          </tbody>
+        </table>
+        <span class="rfc2119-assertion" id="profile-event-payload-size">
+          The size of an event object MUST NOT exceed 64KB, to ensure interoperability with all kinds of consumers.
```

Comment:
where does this come from?

Reply:
A pragmatic limit. 
Why would we have more than 64k for an event payload?

We should discuss the limits together.

Reply:
Similarly, why would need more than 32KB? Or 35KB?

Reply:
64K can be addressed by 16 bits, which is still a limit on some platforms.